pkg/client/client.go

```diff
@@ -633,6 +638,7 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 		fmt.Fprintf(tab, "  Status:\t%s\n", sr.KubeProxyReplacement.Mode)
 		fmt.Fprintf(tab, "  Socket LB:\t%s\n", socketLB)
 		fmt.Fprintf(tab, "  Socket LB Tracing:\t%s\n", socketLBTracing)
+		fmt.Fprintf(tab, "  BPFSocketLBHostnsOnly:\t%s\n", bpfSocketLBHostnsOnly)
```

Comment:
nit, this should better say:

`Socket LB Coverage:` and then spill out either `Host-only` or `Full` for example.

Reply:
Ok. Done.

Reply:
Looks great, thanks @romanspb80 . Could you still add your Signed-off-by to the commit message? If you look intothe git log, you'll see that all commits have it. Then we should be good to move forward and merge. Thanks!